middlewares/authn.js

```diff
@@ -17,6 +17,12 @@ module.exports = function(req, res, next) {
         return;
     }
 
+    if (/^\/$/.test(req.path) && config.authType !== 'none') {
+      // Landing page should not be authenticated unless running locally
```

Comment:
I ended up using this bypass without touching `home.js`, but not so sure if this is valid.